app/pages/resource/resource-page.less

```diff
@@ -1,2 +1,13 @@
 @import 'info/resource-info';
 @import 'reservation-form/reservation-form';
+
+.resource-info {
+  margin-bottom: 40px;
```

Comment:
- [ ] This should probably be inside the .resource-page selector. Just noticed that currently it breaks the search page layout.